src/Bridge/Symfony/Validator/Constraint/Enum.php

```diff
@@ -95,4 +95,22 @@ public function getRequiredOptions()
     {
         return ['class'];
     }
+
+    /**
+     * Fixup deserialized enum instances by replacing them with actual multiton instances,
+     * so strict comparison works.
+     *
+     * {@inheritdoc}
+     */
+    public function __wakeup()
+    {
+        if (!$this->asValue && \is_array($this->choices)) {
+            $this->choices = array_map(function (EnumInterface $enum) {
+                /** @var string|EnumInterface $enumClass */
```

Comment:
wrong?

Reply:
That's only to provide autocompletion. used in multiple places already.

Reply:
I mean the docblcok position, `get_class` returns a string, shouldn't it be under the return below?

Reply:
oh ok got it, strange :)